wraps/python/src/indicators/Qoute.py

```diff
@@ -0,0 +1,11 @@
+from Skender.Stock.Indicators import Quote
+from cstypes import DateTime, Decimal
+
+class Qoute(Quote):
+    def __init__(self, date, open = 0, high = 0, low = 0, close = 0, volume = 0):
```

Comment:
Are these defaults required?  I think `null` would be a better default on the C# handling side.  `0` is an actual viable price value and shouldn't be accidentally applied.

Reply:
You're right. That will be better.
One question, is it null-acceptable in C# module?

Reply:
It will accept an "unset" value natively and it does translate to `null`.  However, I'm not 100% sure what happens when it is set as `null` and converted by the Python wrapper.  Let's give it a try and see what happens.

Reply:
To be clear, the correct spelling is `Quote` and not `Qoute` in both cases, including the file name.

Reply:
Okay, then I'll set default to `None` which is null in Python. 
FYI, when C# module outputs `null`, Python accepts it as `None`.
Then, what we have to check is reverse case.

> To be clear, the correct spelling is Quote and not Qoute in both cases, including the file name.

Oh, sorry, I'll fix that.